scripts/download_artifact.py

```diff
@@ -69,39 +72,44 @@ def download_artifact(artifact_name: str, artifact_id: str, token: str, repo: st
     print(f"download for {artifact_name}: {response.status_code}")
 
     artifact_zip_name = artifact_name.replace(".log", ".zip")
-    artifact_zip = f"./temp/{artifact_zip_name}"
 
+    # output_dir validation
```

Comment:
```suggestion
    # Create output_dir
```
IIUC we aren't validating anything here, just creating the output dir?

Reply:
It was originally meant to validate the existence of output_dir but yes behavior-wise it will create a missing output_dir.